code/modules/surgery/organs/organ_external.dm

```diff
@@ -447,6 +448,10 @@ Note that amputating the affected organ does in fact remove the infection from t
 	if(damage > 15 && local_damage > 30 && prob(damage))
 		cause_internal_bleeding()
 
+/obj/item/organ/external/proc/check_for_burn_wound(damage)
+	if(burn_dam >= min_broken_damage && prob(damage * max(owner.bodytemperature / BODYTEMP_HEAT_DAMAGE_LIMIT, 1)))
```

Comment:
Owner needs to be sanity checked here, an IPC who gets a limb blow off won't have one
[2023-03-30T12:10:56] Runtime in organ_external.dm,452: Cannot read null.bodytemperature